wiki/reverse-proxy.md

```diff
@@ -4,11 +4,11 @@
 
 Apache and mod_proxy should not decode/encode slashes and leave them as they are:
 
-````
+````apacheconfig
 <VirtualHost *:80>
-  AllowEncodedSlashes NoDecode
```

Comment:
Could you apply the same indentation as in the SSL ?

Reply:
Isn't this the syntax for the fenced code? I don't think I changed the indentation of the other part of the code? or are you talking about the tabs between `AllowedEncodedSlashes` and `NoDecode` ?

I took it from https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown

Reply:
@juanpicado Could you answer my question, please? :)